mantidproject/mantid
# Changed step values to update correctly

**Description of work.**

Wavelength and momentum step sizes can be specified as either linear steps of Log steps. The former has dimensions of length and inverse length respectively whilst the Log steps are dimensionless. This PR updates the GUI so that the units displayed change to reflect this when the relevant combo boxes are updated. 

**Report to:** Steve 

**To test:**
 * Open the new SANS GUI  and go to settings-> Q,Wavelength,Detector Limits
 * In the wavelength box change the step type combo box and check that everything else updates appropriately. 
 * In the Q limits box change the 1D step type and check that everything else updates appropriately.


Fixes #23209  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
